src/applications/letters/actions/letters.js

```diff
@@ -22,10 +22,18 @@ import {
   UPDATE_BENEFIT_SUMMARY_REQUEST_OPTION,
   INVALID_ADDRESS_PROPERTY,
   LETTER_HAS_EMPTY_ADDRESS,
+  GET_ENHANCED_LETTERS_DOWNLOADING,
+  GET_ENHANCED_LETTERS_SUCCESS,
+  GET_ENHANCED_LETTERS_FAILURE,
```

Comment:
I felt it was better to add new messaging for the "enhanced" letters object so I could keep my work separate from existing.